Jenkinsfile

```diff
@@ -61,33 +61,30 @@ pipeline {
             }
           }
           steps {
-            withCredentials([string(credentialsId: '52af932f-f13f-429e-8467-e7ff8b965cdb', variable: 'CODECOV_TOKEN')]) {
```

Comment:
This change also has to be made in the Jenkins configuration, because we currently don't run from the Jenkinsfile